components/kyma-environment-broker/internal/process/provisioning/lms_tenant_step.go

```diff
@@ -0,0 +1,101 @@
+package provisioning
```

Comment:
missing test, I know the step is simple and short but all our steps has tests :)

Reply:
The step is tested in TestLmsStepsHappyPath (file: lms_cert_step_test.go). This test combines all 2 lms steps so it tests the LMS requirements.